### PR TITLE
Add authors and fix typo in **Conclusion**

### PR DESCRIPTION
**Pull request by czgdp1807**
I have added @kabra1110 as authors because he fixed issues related to `dict`, `list` and implemented `set` which we are using in this blog post.
Feel free to add more people who have helped in building the features that we are presenting in this blog post.

There was a typo I think in the conclusion which I fixed. And the line `<br/>` doesn't seem to work.

### CODE REVIEW

**Comment by czgdp1807** [comment]
@Smit-create Please feel free to push the changes related to line breaks to my branch. I have added you as a Collaborator on my fork. I don't know, why the `<br>` tag isn't working.

**Comment by Smit-create** [comment]
> @Smit-create Please feel free to push the changes related to line breaks to my branch. I have added you as a Collaborator on my fork. I don't know, why the `<br>` tag isn't working.

I see, locally it runs fine. Need to debug a bit for this.

**Comment by Shaikh-Ubaid** [comment]
>@Smit-create Please feel free to push the changes related to line breaks to my branch. I have added you as a Collaborator on my fork. I don't know, why the <br> tag isn't working.

I think we just need `  ` (two spaces) for line break in markdown. (Please note that if you have `trim white spaces` enabled in `VSCode`, these white spaces get removed. In my case, I temporarily disable the `trim white spaces` feature before saving the file or I open the file in a simple/normal text editor and save it.)

**Comment by Thirumalai-Shaktivel** [comment]
https://stackoverflow.com/a/59374034/15913193

File: content/blog/lpython_mvp.md [hunk 1]
```diff
@@ -848,6 +874,8 @@ Note the optimization flags furnished to each compiler.
 | Python | - |
 
 <br/>
+
```
Comment (Thirumalai-Shaktivel) [inline content/blog/lpython_mvp.md:877]: Yup, this might work, Let's see!

**Comment by Shaikh-Ubaid** [comment]
Is this PR ready?

**Pull request merged**